app/assets/stylesheets/chat.scss

```diff
@@ -552,8 +549,16 @@
   font-weight: bold;
   cursor: pointer;
 
-  img {
+  button {
     opacity: 0.6;
+    border: none;
+    height: 18px;
+    width: 18px;
+    background-size: cover;
+  }
```

Comment:
(refactor/blocking) It looks like the button background is a little funky in Safari:

![screenshot of button showing a gray background behind the three dots](https://user-images.githubusercontent.com/20773163/107245639-48a56600-6a27-11eb-949d-36754ef7df34.png)

I think it can be fixed by just adding an explicit background color:

```suggestion
  button {
    opacity: 0.6;
    border: none;
    height: 18px;
    width: 18px;
    background-size: cover;
    background-color: transparent;
  }
```